src/State.js

```diff
@@ -217,7 +217,7 @@ class State extends EventEmitter {
 		if (config.validator) {
 			var validatorReturn = this.callFunction_(config.validator, [value, name, this]);
 
-			if (validatorReturn instanceof Error) {
+			if (process.env.NODE_ENV === 'developement' && validatorReturn instanceof Error) {
```

Comment:
This `process` variable doesn't exist in browsers, just on node env actually, right? So this wouldn't work everywhere. 

I've noticed that react uses something like this, they use `envify` on their build process to replace these calls, but we don't use `browserify` everywhere, so we couldn't do that. 

I think it's best not to add things like that and just let builds remove `console.error` calls, each on its own way. That way we don't need to depend on something specific, and all builds will work as expected.